config/extension.go

```diff
@@ -28,7 +28,7 @@ type Extensions map[ComponentID]Extension
 
 // ExtensionSettings defines common settings for an extension configuration.
 // Specific processors can embed this struct and extend it with more fields if needed.
-// When embedded in the extension config it must be with `mapstructure:",squash"` tag.
+// When embedded in the extension config, it must be with `mapstructure:",squash"` tag.
```

Comment:
```suggestion
// When embedded in the extension config it must be with `mapstructure:",squash"` tag.
```

Reply:
Same comment as above; I think it would be grammatically correct to have a comma, which may improve readability slightly!